src/ape/managers/accounts.py

```diff
@@ -317,3 +341,20 @@ def __contains__(self, address: AddressType) -> bool:
             any(address in container for container in self.containers.values())
             or address in self.test_accounts
         )
+
+    def use_sender(
+        self,
+        account_id: Union[AccountAPI, AddressType, str, int],
+        autosign: bool,
+        passphrase: Optional[str] = None,
+    ) -> ContextManager:
+        if not isinstance(account_id, AccountAPI):
+            if isinstance(account_id, int) or is_hex(account_id):
+                account = self[account_id]
+            elif isinstance(account_id, str):  # alias
+                account = self.load(account_id)
+        else:
+            account = account_id
+        if isinstance(account, KeyfileAccount):
+            account.set_autosign(autosign, passphrase)
```

Comment:
And then something like this:
```suggestion
        for fn_name, args in kwargs.items():
            getattr(account, fn_name)(*args)
```

Reply:
This will make it more extensible over any account type! (also we won't need to import from `ape_accounts` anymore)

Reply:
Could also do it this way:
```suggestion
       for fn_name, fn_args in kwargs.items():
            if isinstance(fn_args, (tuple, list)):
                getattr(account, fn_name)(*fn_args)
            elif isinstance(fn_args, dict):
                getattr(account, fn_name)(**fn_args)
            else:
                raise  # can't handle `type(fn_args)`
```

Reply:
Yeah, from the point of view of the user of the API, this is strictly worse. And being that ape documentation is nothing to write home about, losing the well-named arguments is just ugh. Not to mention the type information that is lost.

Reply:
I agree with @istankovic, clear kwargs are better than unknown things.

Reply:
I think if that is the case, then the implementer of the plugin should handle the change in this part adding another kwarg if it is needed. As i am not using in the code right now, if i add the kwargs, i need to suppress it also as it wont be used.

Reply:
I am okay with just sticking to `autosign=True` and `unlock=True`, I am still very much against `passphrase="my super secure passphrase that I accidentally commited to github then tried to scrub but it was far too late and now I've lost all my monies"`

Reply:
btw, I think `autosign` sets `unlock=True`, so you might only need to do `set_autosign` if both are present

Reply:
yes, that is why i used just autosign